src/lib/firebaseBackend.js

```diff
@@ -655,6 +655,75 @@ export default class FirebaseBackend {
     }
   }
 
+  // Saves a serviceuser to the database, extra metadata that isn't supported byt the regular firebase user
+  async saveServiceUser(inputFields) {
```

Comment:
@newhouseb let me know what you think of this approach. I don't think the firebase user can hold arbitrary fields such as `contactPreference`, so created this auxiliary collection to store this data.